.gitattributes

```diff
@@ -0,0 +1 @@
+*.xcodeproj/ linguist-generated=true
```

Comment:
Should we be seeing diffs for the `pbxproj` files in this PR?

Should this say **/*.xcodeproj/** instead?